Source/main.cpp

```diff
@@ -111,6 +115,8 @@ void main_main ()
         pp.get("DMI_val",DMI_val);
         pp.get("anisotropy_val",anisotropy_val);
 
+        demag_solver = 0;
```

Comment:
should we make FFT-based demag solver the default?

Reply:
Yeah, that makes sense.  Note that I haven't tested time-advancement yet, just computing a demag field from the initial condition.  So MagneX is still WIP as I continue to work on it.